dev/io.openliberty.security.jakartasec.3.0.internal_fat.config/publish/shared/config/oidcProvider.xml

```diff
@@ -60,6 +60,8 @@
 							https://localhost:${bvt.prop.security_2_HTTP_default.secure}/scopeELUppercaseScopes/Callback,
 							https://localhost:${bvt.prop.security_2_HTTP_default.secure}/scopeELUnknownScope/Callback,
 							https://localhost:${bvt.prop.security_2_HTTP_default.secure}/scopeELDuplicateScope/Callback,
+							https://localhost:${bvt.prop.security_2_HTTP_default.secure}/responseModeQuery/Callback,
+							https://localhost:${bvt.prop.security_2_HTTP_default.secure}/responseModeFormPost/Callback,
 							https://localhost:${bvt.prop.security_2_HTTP_default.secure}/useSessionTrueELTrue/Callback,
```

Comment:
for separate config tests, I'd like to create unique oidcProvider.xml files with unique configs for the OP - this OP is getting pretty overloaded.
Again, this doesn't need updated now, but future tests should do that (and we should come back to this)